Client/Frontend/Browser/Tabs/InactiveTabItemCell.swift

```diff
@@ -0,0 +1,142 @@
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+class InactiveTabItemCell: UITableViewCell, NotificationThemeable, ReusableCell {
+
+    private var viewModel: InactiveTabItemCellModel?
+
+    var selectedView: UIView = {
+        let view = UIView()
+        view.backgroundColor = UIColor.theme.tableView.selectedBackground
+        return view
+    }()
+
+    var leftImageView: UIImageView = {
+        let imgView = UIImageView()
+        imgView.contentMode = .scaleAspectFit
+        imgView.layer.cornerRadius = 5.0
+        imgView.clipsToBounds = true
+        return imgView
+    }()
+
+    var titleLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.textAlignment = .natural
+        label.numberOfLines = 1
+        label.contentMode = .center
+        return label
+    }()
+
+    lazy var bottomSeparatorView: UIView = .build { separatorLine in
+        separatorLine.backgroundColor = UIColor.Photon.Grey40
+    }
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        initialViewSetup()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func configureCell(viewModel: InactiveTabItemCellModel) {
+        self.viewModel = viewModel
+
+        titleLabel.text = viewModel.title
+        titleLabel.font = viewModel.fontForLabel
+        leftImageView.setImageAndBackground(forIcon: viewModel.icon,
+                                            website: viewModel.website) {}
+        separatorInset = UIEdgeInsets(top: 0,
+                                      left: InactiveTabItemCellModel.UX.ImageSize + 2 *
+                                      InactiveTabItemCellModel.UX.BorderViewMargin,
+                                      bottom: 0,
+                                      right: 0)
+        backgroundColor = .clear
+    }
+
+    let containerView = UIView()
+    let midView = UIView()
+
+    private var containerViewLeadingConstraint: NSLayoutConstraint!
+
+    func initialViewSetup() {
+        self.selectionStyle = .default
+        midView.addSubview(titleLabel)
+        containerView.addSubviews(bottomSeparatorView)
+        containerView.addSubview(leftImageView)
+        containerView.addSubview(midView)
+
+        contentView.addSubview(containerView)
+        bringSubviewToFront(containerView)
+
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        bottomSeparatorView.translatesAutoresizingMaskIntoConstraints = false
+        leftImageView.translatesAutoresizingMaskIntoConstraints = false
+        midView.translatesAutoresizingMaskIntoConstraints = false
+        containerView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
We use `.build` to do that automatically for us. So `translatesAutoresizingMaskIntoConstraints = false` is applied automatically. 

Example: 
```
lazy var titleLabel: UILabel = .build { label in
       let label = UILabel()
       label.textColor = .black
       label.textAlignment = .natural
       label.numberOfLines = 1
       label.contentMode = .center
}
```